netebpfext/net_ebpf_ext.c

```diff
@@ -750,19 +798,43 @@ net_ebpf_extension_uninitialize_wfp_components(void)
     NTSTATUS status;
 
     if (_fwp_engine_handle != NULL) {
-        status = FwpmEngineClose(_fwp_engine_handle);
-        if (!NT_SUCCESS(status)) {
-            NET_EBPF_EXT_LOG_NTSTATUS_API_FAILURE(NET_EBPF_EXT_TRACELOG_KEYWORD_EXTENSION, "FwpmEngineClose", status);
-        }
-        _fwp_engine_handle = NULL;
-
+        // Clean up the callouts
         for (index = 0; index < EBPF_COUNT_OF(_net_ebpf_ext_wfp_callout_states); index++) {
             status = FwpsCalloutUnregisterById(_net_ebpf_ext_wfp_callout_states[index].assigned_callout_id);
```

Comment:
should unregister happen after callout delete? Asking because when initializing, we first register callout, and then add callout.

Reply:
Fixed.

Reply:
Fwps* APIs are not dependent on the bfe engine handle. This is not a blocking comment.